rs/nervous_system/integration_tests/tests/sns_ledger_upgrade.rs

```diff
@@ -105,6 +105,8 @@ async fn test_deploy_fresh_sns() {
     // TODO eventually we need to test a swap
 }
 
+// TODO[NNS1-3429]: Re-enable this test after the next SNS-W release.
+#[ignore]
 #[tokio::test]
```

Comment:
The ticket NNS1-3429 is completed, and the test is re-enabled on master. 

I expect the delta under rs/nervous_system to disappear after rebasing this branch.

Reply:
I've rebased this to origin/msater.